src/utils/getFromApiExpectOK.js

```diff
@@ -1,8 +1,13 @@
 import CustomError from './customError';
+import fetchAPI from './fetchAPI';
 
 const getFromApiExpectOK = async (url) => {
-  const response = await fetch(url);
-
+  let response = null;
+  if (url.includes('experiments')) {
```

Comment:
I would go for checking if it is a path and not and url instead of checking for `experiments`, since pretty soon we will want other places, like `data-management`.